tests/unit/recommenders/models/test_rbm.py

```diff
@@ -106,7 +119,7 @@ def check_sampled_values(sampled, s):
     r = Xtr.max()  # obtain the rating scale
 
     # fit the model to the data
-    model.fit(Xtr, Xtst)
+    model.fit(Xtr)
 
     # evaluate the activation probabilities of the hidden units and their sampled values
```

Comment:
can you add a test for the save and load, similar to `def test_class_init(init_rbm):` but create the class, save it, loading and then make sure that you get the right:
```
    # number of hidden units
    assert model.n_hidden == init_rbm["n_hidden"]
    # number of training epochs
    assert model.epochs == init_rbm["epochs"] + 1
    # minibatch size
    assert model.minibatch == init_rbm["minibatch"]
    # keep probability for dropout regulrization
    assert model.keep == init_rbm["keep_prob"]
    # learning rate
    assert model.learning_rate == init_rbm["learning_rate"]
    # standard deviation used to initialize the weight matrix from a normal distribution
    assert model.stdv == init_rbm["init_stdv"]
    # sampling protocol used to increase the number of steps in Gibbs sampling
    assert model.sampling_protocol == init_rbm["sampling_protocol"]
    # number of epochs after which the rmse is displayed
    assert model.display_epoch == init_rbm["display_epoch"]
```

Reply:
Done.